repair/repair.hh

```diff
@@ -69,7 +69,7 @@ std::ostream& operator<<(std::ostream& os, const repair_uniq_id& x);
 
 struct node_ops_info {
     utils::UUID ops_uuid;
-    shared_ptr<abort_source> as;
+    std::vector<shared_ptr<abort_source>> sharded_abort_source;
     std::list<gms::inet_address> ignore_nodes;
     void check_abort();
 };
```

Comment:
Either delete the copy/move constructors and assignment operators or change the type to `std::vector<foreign_ptr<sharerd_ptr<abort_source>>>`. Maybe we should do both to be on the safe side.

Reply:
yikes :)
I'm tempted to simplify the abort design as per https://github.com/scylladb/scylladb/pull/11827#issuecomment-1288404065

Reply:
I wanted to change as little as possible in anticipation for https://github.com/scylladb/scylladb/pull/11597
and then adding support for abort on top of it.

Reply:
I think deleting the move/copy constructors and assignment operators is a small change and it goes a long way in avoiding accidents.